web/src/views/Index.vue

```diff
@@ -122,32 +123,61 @@ export default defineComponent({
     const { t } = useI18n();
     const loading = ref(false);
     const selectedIndexes = ref([]);
+    const filterQuery = ref("");
     const indexes = ref([]);
+    const rowsPerPageOptions = ref([5, 10, 20, 50, 100, 500, 1000]);
```

Comment:
I think `[5,10,20,50,100]` is okay, very big page size is unusual.

Reply:
In env use, the index management function, the most commonly used is batch deletion, I will be deleted once every 2~3 months, with about 10000-20000 indexes records each time. Therefore, it is necessary to keep a big page size.

Reply:
The delete index api support wildcard, you can delete like this: `DELETE /api/index/log-2022*`

Reply:
Okay